pkg/options/simpletemplate/simple_maker.go

```diff
@@ -26,24 +26,21 @@ import (
 
 	bundle "github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/apis/bundle/v1alpha1"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/converter"
-	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/filter"
-	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/maker"
+	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/options"
 )
 
-// Maker makes components via go-templating.
-type Maker struct{}
+// Applier applies options via go-templating.
+type Applier struct{}
 
```

Comment:
why is this public and not private with a public constructor?

Reply:
Sure could. I don't have a strong opinion